decimal.go

```diff
@@ -126,20 +126,28 @@ func Min(first Decimal, rest ...Decimal) Decimal {
 // optimized:
 // New returns a new fixed-point decimal, value * 10 ^ exp.
 func New(value int64, exp int32) Decimal {
+	d, done := tryOptNew(value, exp)
```

Comment:
minor: the Go convention to name the second var here is `ok` (eg. search for "comma ok" [here](https://go.dev/doc/effective_go))

(please update the below usage as well)